<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProgramSchedule</class>
 <widget class="QWidget" name="NewProgramSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start Condition:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_start_condition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Now</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reboot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="combo_repeat_mode">
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interval</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Selection</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Repeat Mode:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_start_time">
       <property name="text">
        <string>Start Time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_start_delay">
       <property name="text">
        <string>Start Delay:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="date_start_time">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1970</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1770</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_repeat_interval">
       <property name="text">
        <string>Repeat Inverval:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_repeat_seconds">
       <property name="text">
        <string>Repeat Seconds:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_repeat_weekdays">
       <property name="text">
        <string>Repeat Weekdays:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_repeat_months">
       <property name="text">
        <string>Repeat Months:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_repeat_days">
       <property name="text">
        <string>Repeat Days:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_repeat_minutes">
       <property name="text">
        <string>Repeat Minutes:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_repeat_hours">
       <property name="text">
        <string>Repeat Hours:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edit_repeat_seconds"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="edit_repeat_minutes"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edit_repeat_hours"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="edit_repeat_days"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="edit_repeat_months"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="edit_repeat_weekdays"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spin_start_delay">
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spin_repeat_interval">
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
